src/pages/sitemap.xml.tsx

```diff
@@ -1,30 +1,2237 @@
+    <?xml version="1.0" encoding="UTF-8"?>
+    <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+    xmlns:news="http://www.google.com/schemas/sitemap-news/0.9"
+    xmlns:xhtml="http://www.w3.org/1999/xhtml"
+    xmlns:image="http://www.google.com/schemas/sitemap-image/1.1"
+    xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
+    <url>
         <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/</loc>
-        <lastmod>2023-05-01</lastmod>
-      </url>
-      <url>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/blog/near-announces-the-blockchain-operating-system</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/inscribe.near/widget/NEAT</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/blog/welcome-to-near-the-b-o-s</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/horizon</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/signin</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/signup</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/nearcon23.near/widget/Index</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/blog/embracing-decentralization-whats-next-for-the-near-wallet</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/inscribe.near/widget/NRC-20</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/Search.IndexPage</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/papers/the-official-near-white-paper</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/ProfilePage</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/ecosystem</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
         <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/NearOrg.HomePage</loc>
-        <lastmod>2023-05-01</lastmod>
-      </url>
-      <url>
-        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/use</loc>
-        <lastmod>2023-05-01</lastmod>
-      </url>
-      <url>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/applications</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/ComponentDetailsPage</loc>
```

Comment:
can be removed. it has no content without proper query params

Reply:
done!